docs/calling-the-asset-delivery-api.md

```diff
@@ -0,0 +1,168 @@
+# Calling the Asset Delivery APIs
+
+## SDK methods
+
+The [Asset Delivery
+API](https://www.sharetribe.com/api-reference/asset-delivery-api.html) allows
+retrieving the data of a single asset or a set of assets by alias or by version.
+The SDK has corresponding methods for each case.
+
+### assetByAlias()
+
+The `assetByAlias()` method retrieves data for a single asset given a version
+alias. It takes an object as its argument and the object must contain:
+
+* `alias`: currently must always have the value `latest`
```

Comment:
Comment: This comment is beyond documentation, but reading this feels a bit weird. I mean, we have a function `assetByAlias` but then the alias must always be `latest`. We likely add support for managing aliases, but that's not coming for a while I guess.

What I'm thinking is that would it make sense to have another function, maybe something like `latestAsset`, or just `asset`, that would call `assetByAlias` with `latest` alias? And if we'd have that, we could leave `assetByAlias` as an internal, non-documented function.